subsys/app_memory/Kconfig

```diff
@@ -0,0 +1,9 @@
+
+menu "General Kernel Options"
+config APP_SHARED_MEM
+	bool
+	prompt "alt application shared memory"
+	default n
+	help
+	 This is a testing option
```

Comment:
Likewise, we'll want better help text.  Maybe "Enable support for selective sharing of global variables among application threads" and include a reference to the doc file?